plugins/module_utils/network/nxos/config/l3_interfaces/l3_interfaces.py

```diff
@@ -482,10 +482,10 @@ def add_commands(self, diff, name=""):
                 self.cmd_order_fixup(commands, name)
         if "ipv6_redirects" in diff:
             # Note: device will auto-disable redirects when secondaries are present
-            if diff["ipv6_redirects"] != self.check_existing(name, "ipv6_redirects"):
-                no_cmd = "no " if diff["ipv6_redirects"] is False else ""
-                commands.append(no_cmd + "ipv6 redirects")
-                self.cmd_order_fixup(commands, name)
+            # if diff["ipv6_redirects"] != self.check_existing(name, "ipv6_redirects"):
```

Comment:
This check must have been added previously for a reason and we should not drop it without a proper justification.